scripts/biomedical/VirtualMetabolicHuman/format_microbes.py

```diff
@@ -33,40 +45,74 @@ def main():
     }
     '''
     result = dc.query(query_str)
-    
-    #wrap the specie_dcids obtained above, in a list
     specie_dcids = ['0'] * len(result)
```

Comment:
nit: this (and below) should be species_dcides?

Reply:
Fixed it, sorry about that!